src/withStyles.jsx

```diff
@@ -109,9 +109,13 @@ export function withStyles(
         // Get and store the result in the stylesFnResultsCache for the component
         // -- not the instance -- so we only apply the theme to the stylesFn
         // once per theme for this component.
-        const cachedResultForTheme = stylesFnResultCacheMap.get(theme);
+        const isClient = typeof window !== 'undefined';
```

Comment:
will this result in a different initial client render, compared to the server render?

Reply:
🤔 I was worried about that. I should test that scenario.